packages/x-components/src/x-modules/tagging/events.types.ts

```diff
@@ -25,4 +28,9 @@ export interface TaggingXEvents {
    * * Payload: The new session duration value.
    */
   SessionDurationProvided: number;
+  /**
```

Comment:
Put the events form A to Z please.

```suggestion
  /**
   * Query tagging info has changed.
   * * Payload: The new query tagging info.
   */
  SearchTaggingEmitted: TaggingInfo;
  /**
   * Session duration has been provided.
   * * Payload: The new session duration value.
   */
  SessionDurationProvided: number;
```